getting-started/cli/new.md

```diff
@@ -20,8 +20,12 @@ Options:
   -d      Select the database database engine, can be one of: pg | mysql | sqlite (default: pg)
   --deps  Installs project dependencies, this is the equivalent of running (shards update)
   -t      Selects the template engine language, can be one of: slang | ecr (default: slang)
+  -r      Use a named recipe.  See documentation at https://amberframework.org.
 ```
 
+See the **Recipes** option of the **Command Line Tool** for information about using
```

Comment:
the sidebar uses an onclick handler to process and load the markdown so I took the easy route rather than spend hours trying to work out how to load another markdown document into the main content area from a markdown link

Reply:
Ouch, that's no fun. I understand why you did that. @faustinoaq do you have any thoughts on how this can be made clickable?

Reply:
I think we need to rewrite the way we manage the guides, because fetching `.md` files and using a hashtag in the link is blocking us from linking sections, I would like to implement something like [ember documentation](https://guides.emberjs.com/v3.0.0/) (maybe using ember a.k.a. Æmber stack :wink: ), although, I think this enhancement would take some time and effort :sweat_smile:   

Meanwhile, I guess we can use some JS hack for manage ajax, history and routes, WDYT?

Reply:
I guess my preference would be that they're generated and served statically, but I'm mostly a backend dev...so all this front end stuff is just magic and sparkles to me.

Reply:
I store my online documentation in the database and this is the ruby script I use to generate my online documentation from markdown files so the content in the database is rendered html.  
```rb
require 'github/markup'
require 'sqlite3'

SQLite3::Database.new( "db/smartbox_development.db" ) do |db|

  db.execute( "delete from posts" )

  Dir.glob("posts/*.md").each do |filename|
    open(filename, "r") do |file|
      text = file.read
      lines = text.split("\n") 
      title = lines.shift
      section = lines.shift
      text = lines.join("\n") 
      body = GitHub::Markup.render_s(GitHub::Markups::MARKUP_MARKDOWN, text)
      db.execute("insert into posts (title,section,body) values(?,?,?)", [title,section,body])
    end
  end

end
```
The documentation is organised into sections on the index page.

Reply:
Oh very nice! I think we can use something similar on amber :+1:

Reply:
If you create an issue to discuss switching to a pre-rendered solution on amberframework.org and the proposal is approved then it is something I could work on.

Reply:
@damianham Done! https://github.com/amberframework/amberframework.org/issues/98

Reply:
Oh, I think we can add a link here now :sweat_smile: 

See: https://amberframework.gitbook.io/amber/getting-started/cli/generate#recipes

![screenshot_20180515_001317](https://user-images.githubusercontent.com/3067335/40037809-c544c11e-57d4-11e8-9dd5-1df15e52ce8a.png)